src/setup/node.rs

```diff
@@ -153,7 +172,10 @@ mod tests {
     async fn start_stop_node() {
         let mut node = Node::new().unwrap();
 
-        node.log_to_stdout(true).start().unwrap();
+        node.log_to_stdout(true)
+            .start(Duration::from_secs(2))
```

Comment:
Suggestion: wondering if we don't just want this to be a global constant, this isn't the only test we'll be waiting on a node to start and I don't really see why the needed duration would change between different tests?

Reply:
Similarly, this can be just inside `start()`. What do you think?

Reply:
Done